src/cz/crcs/ectester/standalone/libs/NettleLib.java

```diff
@@ -0,0 +1,20 @@
+package cz.crcs.ectester.standalone.libs;
+
+import java.security.Provider;
+import java.util.Set;
+
+/**
+ * @author Michal Cech 445431@mail.muni.cz
+ */
+public class NettleLib extends NativeECLibrary {
+
+    public NettleLib() {
+        super("nettle_provider", "hogweed", "gmp");
```

Comment:
This should likely include the nettle lib, it may be okay without it, but may cause issues on some systems.

Reply:
Solved in 81ea9da